lib/stop-tree-cache.js

```diff
@@ -25,8 +25,7 @@ export function create (data, size) {
   }
 
   // build the index and de-delta-code
-  let pixelsRead = 0
-  for (let i = 0, pixel = 0, stopId = 0, time = 0; i < data.length; pixel++) {
+  for (let i = 0, pixel = 0, stopId = 0, time = 0; pixel < size; pixel++) {
     stopTreeCache.index[pixel] = i
     const nStops = read(i++)
```

Comment:
I'm still of the opinion that this function shouldn't be used and the bounds checking should be more explicit.